interop/jvm/src/main/scala/scalaz/zio/interop/catz.scala

```diff
@@ -1,21 +1,27 @@
 package scalaz.zio
 package interop
 
+import scala.util.control.NonFatal
```

Comment:
There is a different definition of `NonFatal` in `RTS`, perhaps that should be exposed somehow, like `def nonFatal` inside `scalaz.zio` package?

Reply:
I'll take a look

Reply:
...and done